elastic/apm-server
# ubi: remove major version

> ⚠️ There is no rush for merging this PR. We can wait for until the new release is out.



## Motivation/summary

to help with the Unified Release contract

It requires changes in the Unified Release too



## Checklist




For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**?
- Is its use being published in **telemetry** to enable product improvement?
- Have system tests been added to avoid regression?

## How to test these changes


Locally or in the CI

For the latter, see [this build](https://buildkite.com/elastic/apm-server-package/builds/805#018bcce4-26be-42eb-b835-7d22eb9bffef/130-154) that produced the new artifacts:

<img width="1140" alt="image" src="https://github.com/elastic/apm-server/assets/2871786/c664d705-9d59-4aa3-9079-8844de847195">


## Related issues
